topics/contributing/tutorials/create-new-slides/tutorial.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: tutorial_hands_on
+
+title: "Creating Slides in Markdown"
+questions:
+  - "How to format slides?"
+  - "How do we add presenter notes?"
+  - "How to use the features of the slide show tool?"
+  - "What sort of content should be included in slides?"
+objectives:
+  - "Create a new set of slides"
+  - "Add presenter comments"
+time_estimation: "15m"
+key_points:
+  - "Slides are often presented before the hands-on portion, provide relevant background information"
+  - "Provide examples that are relevant to your audience"
+  - "Consider writing speaker notes"
+subtopic: slides
+contributions:
+  authorship:
+  - bebatut
+  - hexylena
+  editing:
+  - bgruening
+  - shiltemann
+---
+
+Once we have set up the infrastructure, we are ready to write a slide deck.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+The tutorial's content should be placed in the file `slides.html`. Its syntax and structure are simple, and will have the following structure:
+
+```markdown
+---
+layout: tutorial_slides
+logo: "GTN" # You can also provide an image path, e.g. topics/ai4life/images/AI4Life-logo_giraffe-nodes.png
+
+title: Title of the Slide Deck
+questions:
+- Which biological questions are addressed by the tutorial?
+- Which bioinformatics techniques are important to know for this type of data?
+objectives:
+- The learning objectives are the goals of the tutorial
+- They will be informed by your audience and will communicate to them and to yourself
+  what you should focus on during the course
+- They are single sentences describing what a learner should be able to do once they
+  have done completed tutorial
+- You can use Bloom's Taxonomy to write effective learning objectives
+time_estimation: ''
+key_points:
+- The take-home messages
+- They will appear at the end of the tutorial
+
+contributions:
+  authorship:
+  - contributor1
+  editing:
+  - contributor2
+---
+
+# Slides
+
+- Introduce a subject
+- Quick overview
+- Not too much text
+- Images are Nice
+
+???
+
+- Slides a great way to introduce an audience to your subject.
+
+---
+
+## Another Slide
+
+And some contents
+
+---
+
+## A third slide
+
+More content!
+
+???
+
+- Here go speaker notes
+```
+
+# Metadata
+
+The `slides.html` needs to start with some metadata at the top:
+
+- `layout: tutorial_slides`: please keep the default unless 
+- `title`: title of the tutorial (it will appear on the tutorial page and the topic page)
+- `contributions`: everybody who has contributed to this tutorial (usernames must match those in `CONTRIBUTORS.yaml` file)
```

Comment:
We could point to the metadata description to explain how to fill out that (`authorship`, etc)

Reply:
I did not see it is explained below. Maybe we should remove it there or at least mention it is detailed below (to avoid confusion)